modules/browser/BrowserCapabilities.js

```diff
@@ -66,6 +66,15 @@ export default class BrowserCapabilities extends BrowserDetection {
             || this.isOpera();
     }
 
+    /**
+     * Checks if the current browser is Edge that is Chromium based.
+     */
+    isChromiumBasedEdge() {
```

Comment:
Do we really need to be this explicit? If we are not going to do anything specific to Edge I'd rather see this embedded in `isChromiumBased`. WDYT?